lib/cisco_node_utils/bgp.rb

```diff
@@ -416,13 +416,24 @@ def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
       if match.is_a?(Array)
         return 'false' if match[0] == 'no '
-        return 'size_' + match[1] if match[1]
+        if Utils.nexus_evergreen?
+          return default_event_history_cli unless match[1]
+          # check if the val is size in bytes
+          return match[1] if match[1] =~ /\A\d+\z/
+          return 'size_' + match[1]
+        else
+          return 'size_' + match[1] if match[1]
+        end
       end
       default_event_history_cli
     end
 
     def event_history_cli=(val)
       size = val[/small|medium|large|disable/]
+      if Utils.nexus_evergreen?
```

Comment:
Don't think the `Utils.nexus_evergreen?` check is needed here either.  This can simply be
```
size = val[/small|medium|large|disable|\A\d+\z/]
```